### PR TITLE
Added Minizip

### PR DESCRIPTION
**Pull request by wally4000**
Slightly hackish but it builds and tests complete.

### CODE REVIEW

File: minizip/VITABUILD [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+url="https://github.com/nmoinvaz/minizip"
+source=("https://github.com/nmoinvaz/minizip/archive/2.9.1.zip")
+sha256sums=('8ffe1fdb1d247fc484d1b0b87fdc71f68f3e9ad7c35b3d34a191e00b941df614')
+
+build() {
+ cd minizip-2.9.1
+sed -i '1 i\#define MZ_VERSION_MADEBY_HOST_SYSTEM (MZ_HOST_SYSTEM_UNIX)' mz_os.h
+  mkdir build && cd build
+cmake .. -DCMAKE_TOOLCHAIN_FILE=$VITASDK/share/vita.toolchain.cmake -DCMAKE_INSTALL_PREFIX=$prefix 
+  make
```
Comment (devnoname120) [inline minizip/VITABUILD]: `make -j$(nproc)`

File: minizip/VITABUILD [hunk 2]
```diff
@@ -0,0 +1,19 @@
+pkgname=minizip
+pkgver=2.9.1
+pkgrel=1
+url="https://github.com/nmoinvaz/minizip"
+source=("https://github.com/nmoinvaz/minizip/archive/2.9.1.zip")
+sha256sums=('8ffe1fdb1d247fc484d1b0b87fdc71f68f3e9ad7c35b3d34a191e00b941df614')
+
+build() {
+ cd minizip-2.9.1
```
Comment (devnoname120) [inline minizip/VITABUILD]: $pkgver

File: minizip/VITABUILD [hunk 3]
```diff
@@ -0,0 +1,19 @@
+pkgname=minizip
+pkgver=2.9.1
+pkgrel=1
+url="https://github.com/nmoinvaz/minizip"
+source=("https://github.com/nmoinvaz/minizip/archive/2.9.1.zip")
```
Comment (devnoname120) [inline minizip/VITABUILD]: $pkgver

File: minizip/VITABUILD [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+build() {
+ cd minizip-2.9.1
+sed -i '1 i\#define MZ_VERSION_MADEBY_HOST_SYSTEM (MZ_HOST_SYSTEM_UNIX)' mz_os.h
+  mkdir build && cd build
+cmake .. -DCMAKE_TOOLCHAIN_FILE=$VITASDK/share/vita.toolchain.cmake -DCMAKE_INSTALL_PREFIX=$prefix 
+  make
+}
+
+package () {
+  cd minizip-2.9.1/build
```
Comment (devnoname120) [inline minizip/VITABUILD]: $pkgver

**Comment by Rinnegatamante** [comment]
Bump @wally4000

**Comment by wally4000** [comment]
Issues fixed. Cheers

**Comment by Rinnegatamante** [comment]
Seems like minizip build's missing. Please add it here: https://github.com/vitasdk/packages/blob/master/travis_packages.sh

**Pull request merged**